modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc

```diff
@@ -1,19 +1,7 @@
-[id='proc-add-custom-app-file-openshift_{context}']
-= Adding a custom application configuration file to {ocp-brand-name}
+[id='proc-add-custom-app-file-openshift-helm_{context}']
```

Comment:
```suggestion
[id="proc-add-custom-app-file-openshift-helm_{context}"]
```
All the headings should be using " " instead of ' ' per the mod docs guidelines and templates